qa/rpc-tests/importprunedfunds.py

```diff
@@ -5,7 +5,7 @@
 
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import *
-import decimal
+
```

Comment:
you could have had an all-red diff if you didn't add empty lines to balance it out :)